app/views/posts/show.html.erb

```diff
@@ -1 +1,14 @@
-<h1>Details of a selected post<h1>
\ No newline at end of file
+<h1>Blog App</h1>
+<article class='no-posts'>
+  <div class='post-title'>
+    <h1><%=@post.title%> by <%=@post.author.name%></h1>
+    <div class='post-count-best'>Comments: <%=@post.comment_counter%>, Likes: <%=@post.likes_counter%></div>
+  </div>
+  <div class = 'post-kont'><%=@post.text%></div>
+  <ul class='comment-content'>
+  <% @post.comments.each do |comment| %>
+    <li><%= comment.author.name %>: <%= comment.text %></li>
+  <% end %>
+  <%= 'No comments' if @post.comments.empty? %>
+</ul>
+</article>
```

Comment:
Nice effort for making the requested changes. However, kindly check the border in the posts show page. The first box/border contains the post title, comments and likes as well as the message/post body. Please update it to match the given wireframe as shown below. 

### Your design 

![Screenshot from 2023-08-31 00-58-31](https://github.com/emmanuelsteven/MyBlogApp/assets/65233860/f42cb5c4-c184-44dc-8ef2-55a2629201d5)


### Actual design 

![Screenshot from 2023-08-31 01-00-56](https://github.com/emmanuelsteven/MyBlogApp/assets/65233860/39711565-f91c-43fc-9d6b-f72f188f69c1)